taggit/forms.py

```diff
@@ -52,3 +52,11 @@ def has_changed(self, initial_value, data_value):
         initial_value.sort()
 
         return initial_value != data_value
+
+
+class MergeTagsForm(forms.Form):
+    new_tag_name = forms.CharField(
+        label="New Tag Name",
+        max_length=255,
```

Comment:
if you want to test your validation, protip: set `max_length` to `2` temporarily and you should be able to more easily see errors.

Reply:
I tested this earlier and the max_length works, but it shouldn't throw an error at all because the form input box will be be limited to the max_length. For instance when I set it to 2 and try to add a tag "fruit" it will stop at "fr" and not let me input any other characters after that.